src/test/storage/dictionary_column_test.cpp

```diff
@@ -53,5 +54,67 @@ TEST_F(StorageDictionaryColumnTest, LowerUpperBound) {
   EXPECT_EQ(dict_col->lower_bound(15), opossum::INVALID_VALUE_ID);
   EXPECT_EQ(dict_col->upper_bound(15), opossum::INVALID_VALUE_ID);
 }
+TEST_F(StorageDictionaryColumnTest, attribute_vector) {
+  auto col = opossum::make_shared_by_column_type<opossum::BaseColumn, opossum::DictionaryColumn>("string", vc_str);
+  auto dict_col = std::dynamic_pointer_cast<opossum::DictionaryColumn<std::string>>(col);
+}
+
+TEST_F(StorageDictionaryColumnTest, _assign_attribute_vector) {
+  std::shared_ptr<opossum::ValueColumn<std::string>> vc_str = std::make_shared<opossum::ValueColumn<std::string>>();
+  vc_str->append("Bill");
+  auto col = opossum::make_shared_by_column_type<opossum::BaseColumn, opossum::DictionaryColumn>("string", vc_str);
```

Comment:
Vermutlich könnte man den ganzen Testcase - wie die anderen auch - in den `opossum`-Namespace legen und sämtliche sonstige Verweise auf den Namensraum innerhalb des Testcases streichen.